Frontend/src/const.js

```diff
@@ -1,7 +1,7 @@
-const port = 61348;
-const route = `http://sso.julianjp.com:8080/api`;
+const dev_port = 61348;
+const dev_route = `http://localhost:${dev_port}/api`;
 
 export default {
-    API_PORT: port,
-    API_ROUTE: route
+    DEV_PORT: dev_port,
+    DEV_ROUTE: dev_route
```

Comment:
Don't revert this file. These need to come from env variables, so either make that change or leave as-is for merge.